runelite-client/src/main/java/net/runelite/client/plugins/chathistory/ChatHistoryPlugin.java

```diff
@@ -42,12 +43,18 @@
 {
 	private static final String WELCOME_MESSAGE = "Welcome to RuneScape.";
 	private static final Set<ChatMessageType> ALLOWED_HISTORY = Sets.newHashSet(
-		ChatMessageType.PUBLIC,
-		ChatMessageType.CLANCHAT,
-		ChatMessageType.PRIVATE_MESSAGE_RECEIVED,
-		ChatMessageType.PRIVATE_MESSAGE_SENT,
-		ChatMessageType.PRIVATE_MESSAGE_RECEIVED_MOD,
-		ChatMessageType.GAME
+			ChatMessageType.PUBLIC,
+			ChatMessageType.CLANCHAT,
+			ChatMessageType.PRIVATE_MESSAGE_RECEIVED,
+			ChatMessageType.PRIVATE_MESSAGE_SENT,
+			ChatMessageType.PRIVATE_MESSAGE_RECEIVED_MOD,
+			ChatMessageType.GAME
+	);
+
+	private static final Set<ChatMessageType> PRIVATE_MESSAGE_TYPE = Sets.newHashSet(
```

Comment:
Maybe concant the allowed history during creating with PRIVATE_MESSAGE_TYPE, so the info would not be duped here?